docs/developer-docs/functionality/bitcoin/local-development.md

```diff
@@ -0,0 +1,292 @@
+        rpcauth=ic-btc-integration:cdf2741387f3a12438f69092f0fdad8e$62081498c98bee09a0dce2b30671123fa561932992ce377585e8e08bb0c11dfa
+
+5. Run `bitcoind` to start the bitcoin client using the following command:
+
+        ./bin/bitcoind -conf=$(pwd)/bitcoin.conf -datadir=$(pwd)/data --port=18444
+
+    If everything went well, you should see the following output. `bitcoind` is now ready to accept requests.
+
+        2022-07-11T12:49:51Z Bitcoin Core version v22.0.0 (release build)
+        ...
+        2022-07-11T12:49:51Z Config file arg: regtest="1"
+        2022-07-11T12:49:51Z Config file arg: rpcauth=****
+        2022-07-11T12:49:51Z Config file arg: rpcpassword=****
+        2022-07-11T12:49:51Z Config file arg: rpcuser=****
+        ...
+        2022-07-11T12:49:51Z Bound to 127.0.0.1:18445
+        2022-07-11T12:49:51Z Bound to [::]:18444
+        2022-07-11T12:49:51Z Bound to 0.0.0.0:18444
+        ...
+        2022-07-11T12:49:52Z opencon thread start
+        2022-07-11T12:49:52Z addcon thread start
+        2022-07-11T12:49:52Z 0 addresses found from DNS seeds
+        2022-07-11T12:49:52Z dnsseed thread exit
+
+    :::note
+    The command above assumes that port `18444` on your machine is available. If it isn't, change the port specified in `--port`
+    accordingly.
+    :::
+
+## Deploying Your Bitcoin Dapp
+
+Now that you have a local Bitcoin network, you're ready to start
+developing and locally testing your Bitcoin dapps.
+
+For the next steps, we'll be leveraging the example project
+in the [examples repo](https://github.com/dfinity/examples) to showcase some of the key concepts. Let's clone it and explore
+what it's like to deploy a Bitcoin dapp locally.
+
+1. Clone the `examples` repo
+
+        git clone https://github.com/dfinity/examples
+
+2. Go to the `basic_bitcoin` example in the language of your choice
+
+        # For motoko
+        cd examples/motoko/basic_bitcoin
+
+        # For rust
+        cd examples/rust/basic_bitcoin
+
+3. If you're on a Mac, install [Homebrew](https://brew.sh/) and then run the following to install additional packages.
+
+        brew install llvm binaryen cmake
+
+### Configuring `dfx.json`
+
+For Bitcoin projects, you need to configure `dfx` to connect to your local Bitcoin node.
+You already configured your local Bitcoin node to accept connections on port `18444`,
+so in `dfx.json` you can add the following configuration in the `defaults` section:
+
+    "bitcoin": {
+        "enabled": true,
+        "nodes": ["127.0.0.1:18444"]
+    }
+
+The example code already includes this in its `dfx.json`.
+
+### Deploying in `regtest` Mode
+
+Your local Bitcoin node operates in what's called "regression testing mode", or [regtest mode](https://developer.bitcoin.org/examples/testing.html#regtest-mode).
+You can now deploy your canister and configure it to connect to your local `Regtest` network.
+
+1. Run `dfx start`.
+
+    :::tip
+    If when running `dfx start` and you see errors like
```

Comment:
The sentence looks a bit weird to me due to the "and" presence.

Reply:
Fixed.